client/settings/express-checkout-settings/platform-checkout-settings.js

```diff
@@ -38,69 +38,84 @@ const PlatformCheckoutSettings = () => {
 
 	return (
 		<div className="platform-checkout-settings">
-			<Card>
-				<CardBody>
-					<CheckboxControl
-						checked={ isPlatformCheckoutEnabled }
-						onChange={ updateIsPlatformCheckoutEnabled }
-						label={ __( 'Enable WooPay', 'woocommerce-payments' ) }
-						help={ __(
-							'When enabled, customers will be able to checkout using WooPay',
-							'woocommerce-payments'
-						) }
-					/>
-				</CardBody>
-			</Card>
-			<Card style={ { marginTop: 12 } }>
-				<div className="platform-checkout-settings__preview">
-					<PlatformCheckoutPreview
-						storeName={ wcSettings.siteTitle }
-						storeLogo={ platformCheckoutStoreLogo }
-					></PlatformCheckoutPreview>
-					<div className="platform-checkout-settings__preview-fade"></div>
-				</div>
-				<CardBody>
-					<div className="platform-checkout-settings__custom-message-wrapper">
-						<h4>
-							{ __(
-								'Store logo on checkout',
+			{ 'general' === section && (
```

Comment:
Maybe we should split this component into two others instead of using a conditional to return the section wanted. What do think?

Reply:
It's the same approach already used on `Apple Pay / Google Pay` settings [here](https://github.com/Automattic/woocommerce-payments/blob/fix/4157-show-woopay-error-message/client/settings/express-checkout-settings/payment-request-settings.js#L161) and [here](https://github.com/Automattic/woocommerce-payments/blob/fix/4157-show-woopay-error-message/client/settings/express-checkout-settings/index.js#L51).

Reply:
ok. That looks ugly to me. But as it's already a pattern let's keep it for now